android/app/src/main/java/com/reactnativesampleapp/OnfidoSDK.java

```diff
@@ -58,7 +64,10 @@ public String getName() {
     }
 
     @ReactMethod
-    public void startSDK(Callback successCallback, Callback errorCallback) {
+    public void startSDK(String token, String applicantId, Callback successCallback, Callback errorCallback) {
+
+        System.out.printf(String.format("token=%s, applicantId=%s", token, applicantId));
```

Comment:
Can we remove this? Logging tokens is not a good idea

Reply:
this added for debugging but forgot to remove when cleaning up